curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/6434759f78ec812264ff8f34.md

```diff
@@ -49,6 +49,12 @@ Your callback function should be empty.
 assert.match(code, /letters\.forEach\(\s*\(?\s*letter\s*\)?\s*=>\s*\{\s*\}/)
 ```
 
+Your callback function should be nested in `range(1, 99)`'s forEach callback
+
+```js
+assert.match(code, /letters\.forEach\(\s*\(?\s*letter\s*\)?\s*=>\s*\{\s*\}.*\s*\}\)/)
```

Comment:
This checks only if there's _nesting_, a bit safer is making sure it's within the `range(1, 99).forEach(number => {})`
```suggestion
assert.match(code, /range\s*\(\s*1\s*,\s*99\s*\)\s*.forEach\s*\(\s*(\(\s*number\s*\)|number)\s*=>\s*\{\s*[^}]*letters\.forEach\(\s*(\(\s*letter\s*\)|letter)\s*=>\s*\{\s*\}\s*\)\s*\}\s*\)/)
```